robot_driver/src/robot_driver.cpp

```diff
@@ -67,6 +69,17 @@ RobotDriver::RobotDriver(ros::NodeHandle nh, int argc, char **argv) {
                            sit_joint_angles_);
   quad_utils::loadROSParam(nh_, "robot_driver/torque_limit", torque_limits_);
 
+  // load Comp_filter params
+  quad_utils::loadROSParam(nh_, "robot_driver/high_pass_a", high_pass_a_);
+  quad_utils::loadROSParam(nh_, "robot_driver/high_pass_b", high_pass_b_);
+  quad_utils::loadROSParam(nh_, "robot_driver/high_pass_c", high_pass_c_);
+  quad_utils::loadROSParam(nh_, "robot_driver/high_pass_d", high_pass_d_);
+
+  quad_utils::loadROSParam(nh_, "robot_driver/low_pass_a", low_pass_a_);
+  quad_utils::loadROSParam(nh_, "robot_driver/low_pass_b", low_pass_b_);
+  quad_utils::loadROSParam(nh_, "robot_driver/low_pass_c", low_pass_c_);
+  quad_utils::loadROSParam(nh_, "robot_driver/low_pass_d", low_pass_d_);
```

Comment:
Same as above, could these be in init for just the CF? May need to do a dynamic cast in cast state estimator isn't type CF, see leg controller for examples of this.